### DISCUSSION TITLE
Zigbee devices incorrect state

### DISCUSSION THREAD

**Discussion by rvdk92** [Koenkk/zigbee2mqtt#12981] [Devices]
Hello,

I have couple of problems maybe is there anybody with suggestions for me.

Setup: 
HP EliteDesk i5
Proxmox
HomeAssistant OS
Zigbee2MQTT 1.25.0-dev
ConBee2 (latest firmware)
Zigbee2MQTT channel 11 (Wireless Accespoints channel 1, 6 and 9) and i have no neighbors
3 meter extentions usb cable between computer and ConBee2

Previous setup with same issues (Raspberry 4B with same ConBee2 stick.)

The biggest problem is that HomeAssistant does not match the actual status of the device.
So in my HomeAssistant I see that a device is on while it is off. I click what I want, but nothing happens.
For my power plugs I can solve this by going to Zigbee2MQTT and clicking on the device refresh status.
But I can't click refresh on other devices such as temperature sensors or motion sensors and they don't work at all for a long time. only if i reset them completely.
And a lot of plugs TuYa TS011F doesn't show the power or current state if they are on.

Device wise it's a mix.. here's an output of the log from zigbee2mqtt.. anyone know what I can try?

What I've tried so far, installed new versions several times of latest home assistant, zigbee2mqtt, reconnected devices, moved computer with conbee in the room. But then it's fine for 2 days and then problems again.

I hope someone can help me.

Output log:

Zigbee2MQTT:error 2022-06-28 22:05:50: Publish 'set' 'state' to 'Lampen woonkamer boekenkast' failed: 'Error: Command 0xa4c138f708f9d052/1 genOnOff.on({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-28 22:54:54: Publish 'set' 'state' to 'Led strip TV' failed: 'Error: Command 0xa4c138efd4094e94/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-28 22:55:14: Publish 'set' 'state' to 'Led strip TV' failed: 'Error: Command 0xa4c138efd4094e94/1 genOnOff.on({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-28 22:55:54: Publish 'set' 'state' to 'Nachtlampje Robbert' failed: 'Error: Command 0xa4c1380a214afbe7/1 genOnOff.on({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-28 22:55:54: Publish 'set' 'state' to 'Nachtlampje Nanette' failed: 'Error: Command 0xa4c1384ac2ec0d37/1 genOnOff.on({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-28 22:55:54: Publish 'set' 'state' to 'Lamp woonkamer links naast tv' failed: 'Error: Command 0xa4c138f4ef80d753/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-28 22:55:54: Publish 'set' 'state' to 'Lampen woonkamer boekenkast' failed: 'Error: Command 0xa4c138f708f9d052/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-29 09:00:46: Publish 'set' 'state' to 'Lampen woonkamer boekenkast' failed: 'Error: Command 0xa4c138f708f9d052/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-29 10:08:01: Publish 'set' 'state' to 'Lampen woonkamer boekenkast' failed: 'Error: Command 0xa4c138f708f9d052/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:error 2022-06-29 11:03:18: Publish 'set' 'state' to 'Lampen woonkamer boekenkast' failed: 'Error: Command 0xa4c138f708f9d052/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'

**Comment by ConorIA** [comment]
I'm afraid I don't have anything to say to help you, only to comisserate. 

Are the smart plugs acting as routers in your network? I recently added a Sonoff S40ZBTPB Lite smart plug to boost the signal to some of my outdoor smart bulbs and ever since home assistant is totally confused. Devices appear as on when they're off or as unavailable when Zigbee2MQTT can see them just fine. I am running on the dev docker image to try to avoid https://github.com/Koenkk/zigbee2mqtt/issues/13382, but it is still no good. 

I'm not sure, but I think things start out fine, but then eventually states go out of sync. I wonder if forcing a periodic restart of Zigbee2MQTT is necessary.
